securedrop/tests/test_journalist.py

```diff
@@ -459,6 +460,24 @@ def test_admin_add_user_without_username(self):
                                           is_admin=False))
         self.assertIn('Missing username', resp.data)
 
+    @patch('journalist.app.logger.error')
+    @patch('journalist.Journalist')
```

Comment:
`@patch('journalist.Journalist', side_effect=IntegrityError('STATEMENT', 'PARAMETERS', None))` should work here and you can get rid of lines 469-471 below.